lib/linguist/languages.yml

```diff
@@ -3917,6 +3917,18 @@ Pike:
   tm_scope: source.pike
   ace_mode: text
   language_id: 287
+PlantUML:
+  type: markup
+  color: "#CEF6CE"
+  extensions:
+  - ".puml"
+  - ".iuml"
```

Comment:
Are [these files](https://github.com/search?p=1&q=extension%3Aiuml+NOT+startuml+NOT+enduml&type=Code) also PlantUML? They don't look like the other files.

Reply:
Some are, some arent.

`.iuml` seems to be used for plantuml includes, like this:
https://github.com/weedySeaDragon/plantuml-documentation/blob/6365c96a35eefb635796d71a20404b80d50d3e21/plantuml-ae-style.iuml

It also seems to be used for some other sort of include for some other UML tool which I'm not familiar with.

You could recognize the plantuml includes by looking for skinparam or !include or !define directives.

Reply:
Actually looking at a lot of these, many of them could be plantuml

Reply:
Same as above, for `.pu` and `.wsd`.

Reply:
I've checked again the `iuml` files and as far as I can tell they are all valid plantuml; I thought it was some other UML format but it is just used for plantuml includes. I haven't been able to find any that *aren't* valid plantuml as yet - so I've kept that extension for now.

Reply:
Ok. Let's keep it that way then. We can always resolve issues later if they arise.

Reply:
If these files are also valid PlantUML, then it would be good to add one or two of them as samples. Won't be useful for the classification for now, but it would be good to have for tests.